docs/app-portal-oss.mdx

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
I think it should be nested under the `app-portal` name, so live in https://docs.svix.com/app-portal/oss.